### PR TITLE
Set bot in 2P

### PR DESCRIPTION
**Pull request by elsie43**
- You can control the player with **WASD** now

### CODE REVIEW

File: src/main/player/Player.java [hunk 1]
```diff
@@ -30,55 +30,51 @@ public Player(int x, int y, String player, String imageName, boolean ifBot) {
...
-            else 
-                slope = speed_y/speed_x;
-            
-            //if ball is in player2's area and the ball is dropping to player1
-            if (speed_y > 0 && ball_x < 650) { 
+            else
+                slope = speed_y / speed_x;
+
+            // ball is in bot's area and dropping
+            if (speed_y > 0 && ball_x > 630) { // 630 = stick_x - 10 (-10 for judging earlier)
```
Comment (arasHi87) [inline src/main/player/Player.java]: use `Content.STICK_X` calculate instead of use magic number

File: src/main/player/Player.java [hunk 2]
```diff
@@ -87,9 +83,13 @@ public void moveByBall(int ball_x, int ball_y, float speed_x, float speed_y){
                 dx = 0;
             }
 
-            if(ball_x > 500 && Math.abs(predict_x - hit_x) < 15)
+            /**
+             * if ball in opponent's area, let the bot move to the middle of the area of
+             * itself and stay there
+             */
+            if (ball_x < 620 && Math.abs(predict_x - hit_x) < 15)
                 dx = 0;
```
Comment (arasHi87) [inline src/main/player/Player.java]: same as above problem

File: src/main/player/Player.java [hunk 3]
```diff
@@ -62,12 +62,12 @@ public void moveByBall(int ball_x, int ball_y, float speed_x, float speed_y) {
                 slope = speed_y / speed_x;
 
             // ball is in bot's area and dropping
-            if (speed_y > 0 && ball_x > 630) { // 630 = stick_x - 10 (-10 for judging earlier)
+            if (speed_y > 0 && ball_x > (stick_x - 10)) {
                 predict_x = (580 - ball_y + slope * ball_x) / slope;
```
Comment (arasHi87) [inline src/main/player/Player.java]: Your misunderstanding what I mean, I mean there got a constant `STICK_X` in `main.Content`(you can see [here](https://github.com/arasHi87/MeowBall/blob/master/src/main/Content.java#L7)) that what u can use xD
But whatever, this work too, feel free to use any solution u like

Comment (elsie43) [reply src/main/player/Player.java]: ohhhhhh.... I didn't notice that lol 
Fixed it already !

**Pull request merged**